src/index.html

```diff
@@ -148,7 +159,72 @@ <h2 class="modal__title">New Post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script defer src="./scripts/index.js"></script>
-    <script defer src="./scripts/validation.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button
+          type="button"
+          class="modal__close-btn"
+          aria-label="Close modal"
+        ></button>
+        <h2 class="modal__title">Edit avatar</h2>
+        <form
+          name="edit-profile"
+          class="modal__form"
+          id="edit-avatar-form"
+          novalidate
+        >
+          <label for="profile-avatar-input" class="modal__label">
+            Avatar
+            <input
+              id="profile-avatar-input"
+              name="avatar"
+              class="modal__input"
```

Comment:
* Please, validate the avatar input as `url` https://snipboard.io/rje03v.jpg 


`type="url"` should be here